metrics/run_all_metrics.sh

```diff
@@ -56,6 +56,14 @@ EOF
 function run_latency_tests() {
 	# Run the time tests
 	bash ${SCRIPT_PATH}/time/docker_workload_time.sh true busybox $RUNTIME 100
+	# Launch time for first container - no scaling, with network
+	bash ${SCRIPT_PATH}/time/launch_times.sh -i ubuntu -n 100
```

Comment:
uuhmm seems like you forget to use ```-r``` option

Reply:
:-) Good spot, but it was deliberate. You'll see that almost none of the other metrics calls in this file set the `RUNTIME` either - it gets set implicitly to a default in the `common.sh` https://github.com/clearcontainers/tests/blob/master/metrics/lib/common.bash#L23 
So, `-r` is really sort of redundant for this script I guess, as it (like most of the other scripts) will pick up `RUNTIME` from the environment.
For now, I'll leave `-r` as an option unless somebody objects - it's doing no harm, and we'd have to do another push/CI/review cycle ;-)